ACMAS/app/ACMAS_Web/search.py

```diff
@@ -151,13 +156,16 @@ def getFiles(self, course, fType):
         Returns:    QuerySet of Question
         """
         if fType == "All":
+            # return UploadedFile.objects.filter(Course=course)
             return UploadedFile.objects.filter(course=course)
         else:
+            # return UploadedFile.objects.filter(Course=course, Flag=fType)
             return UploadedFile.objects.filter(course=course, flag=fType)
 
     def getPath(self, name):
         """
         Parameters: String name - name of file to get path of
         Returns:    String of file directory of file
         """
+        # return UploadedFile.objects.get(Name=name).file_dir
         return UploadedFile.objects.get(name=name).file_dir
```

Comment:
Is there a case where the `.get()` won't return and `.file_dir` will fail?

Reply:
This would likely be impossible behavior since any uploaded file automatically has a path assigned by Django. Are you thinking of a case where a file is deleted/no longer exists but still has a value in the db?

Reply:
Or if it's an invalid file name, then the `.get()` might return None and there will be error?